src/OrchardCore.Modules/OrchardCore.Flows/wwwroot/Scripts/flows.edit.js

```diff
@@ -1,7 +1,10 @@
 var createEditorUrl = $('#buildEditorUrl').attr("value");
 var widgetTemplate = function (data, prefixesName, prefix, contentTypesName, contentType) {
-    return '<div class="widget-template">' + data + '<input type="hidden" name="' + prefixesName + '" value="' + prefix + '" /><input type="hidden" name="' + contentTypesName + '" value="' + contentType + '" /></div>';
+    return '<div class="widget-template col-md-12">' + data + '<input type="hidden" name="' + prefixesName + '" value="' + prefix + '" /><input type="hidden" name="' + contentTypesName + '" value="' + contentType + '" /></div>';
```

Comment:
`col-md` is sufficient.

Reply:
Just to stay consistent with all  col sizes (col-md-4,col-md-6 etc)  used for Flowmetada used col-md-12

Reply:
You don't need to specify 12 as col is by default 12

Reply:
I meant, if we use only col, col-md the new widget would adjust in remaining space within same row, but it's Metadata value will be 100 in radio input, if user saves it then it will be saved in for next row

Means UI state becomes inconsistent between before save and after save